### ISSUE TITLE
How to set verify_peer false?

### ISSUE THREAD

**Issue by tobsn** [fguillot/picoFeed#304]
```
stream_context_create([
	'ssl' => [
		'verify_peer' => false,
		'verify_peer_name' => false
	]
]);
```

How would I set this with picoFeed? Thanks!

**Comment by BernhardPosselt** [comment]
You don't because you want to verify certificates.

**Comment by schester44** [comment]
@tobsn Did you ever figure this out? I could use this as well, getting cURL error 77.

**Comment by tobsn** [comment]
@schester44 nope, couldn't find a good solution, used file_get because for the purpose i needed it it was just fetching a news feed so not really that important :)
